tests/test_models.py

```diff
@@ -213,9 +212,216 @@ def test_wishlist_id_is_increment(self):
 
         wishlist = WishlistFactory()
         wishlist.create()
-        expected = wishlist.wishlist_id + 1
+        expected = wishlist.id + 1
 
         for wishlist in WishlistFactory.create_batch(5):
             wishlist.create()
-            self.assertEqual(wishlist.wishlist_id, expected)
+            self.assertEqual(wishlist.id, expected)
             expected += 1
+
+
+#####################
+# WishistItem Tests
+#####################
+class TestWishlistItem(unittest.TestCase):
+    """Test cases for WishlistItem Model"""
+
+    serialized = {}
+
+    @classmethod
+    def tearDownClass(cls):
+        """This runs once after the entire test suite"""
+        db.session.query(WishlistItem).delete()
+        db.session.commit()
+
+    def setUp(self):
+        """This runs before each test"""
+        db.session.query(WishlistItem).delete()
+        db.session.commit()
+
+        self.serialized = {
+            "id": 1,
+            "wishlist_id": 2,
+            "product_id": 3,
+            "product_name": "DevOps for Dummies",
+            "product_price": 29.99,
+            "quantity": 2,
+            "created_date": datetime.strptime(
+                "2023-10-12 00:00:00", "%Y-%m-%d %H:%M:%S"
+            ),
+        }
```

Comment:
I'm unsure where the self.serialized is used and if it's needed for every test.

Reply:
Yeah it's not needed for every test @studentgrad0419.  But I use it in a couple tests (test_wishlist_item_deserialize and  test_wishlist_item_deserialize_with_invalid_input). I thought it might be needed for others as well, but in the end just used for 2.  I think I'll leave as is for now, but concede it is a teeny, tiny bit inefficient!  Thanks for the careful review!